source/deployment/desktop-app-deployment.rst

```diff
@@ -0,0 +1,112 @@
+
+  Description
+    List of server URLs and their display names added to the desktop app by default, which the user cannot modify. Users can still add servers `through the Server Management page <https://docs.mattermost.com/help/apps/desktop-guide.html#server-management>`_ unless ``enableServerManagement`` is set to false. 
+    
+    Expects an array of key-value pairs.
+
+  Example
+    .. code-block:: none
+
+      defaultTeams: [
+        {
+          name: 'example',
+          url: 'https://example.com'
+        },
+        {
+          name: 'mattermost',
+          url: 'https://www.mattermost.com'
+        }
+      ]
+
+`helpLink`
+~~~~~~~~~~~~~~~~
+
+  Description
+    The URL of the help documentation in Help > Learn More menu bar item. If none specified, the menu option is hidden.
+    
+    Expects a string.
+
+  Examples
+    .. code-block:: none
+
+      helpLink: 'https://about.mattermost.com/default-desktop-app-documentation/'
+      helpLink: ''
+
+`enableServerManagement`
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+  Description
+    Controls whether users can add, edit or remove servers on the app settings page. If set to false, at least one server must be specified for ``defaultTeams`` or else users cannot interact with any servers.
+    
+    Expects a boolean, true or false.
+
+  Examples
+    .. code-block:: none
+
+      enableServerManagement: true
+
+2. To build the application, follow the `Mattermost Desktop Development Guide <https://github.com/yuya-oc/desktop/blob/master/docs/development.md>`_.
+
+Windows App: Pre-Configuration and Silent Deployment
+------------------------------------------------------
+
+You can distribute the official Windows Desktop App silently to end users, pre-configured with the server URL and other app settings.
+
+1. Download the latest Windows installer from the `Mattermost download page <https://about.mattermost.com/download/#mattermostApps>`_.
+
+2. Move the executable file into a shared folder.
+
+3. Create a batch file in Windows
+
+  1. Open a text editor of your choice, such as Notepad or Notepad++.
+  2. Copy and paste the following commands in the text file
+
+    .. code-block:: none
+
+
+      rem "Step 1: Install mattermost desktop into user's local disk"
+      start /wait \\SERVER\shared_folder\mattermost-setup-3.7.0-win64.exe --silent
+
+      rem "Step 2: Generate initial config.json into user's config directory"
+      (
+        echo {
+        echo   "version": 1,
+        echo   "teams": [
+        echo     {
+        echo       "name": "pre-release",
+        echo       "url": "https://pre-release.mattermost.com/core"
+        echo     }
+        echo   ],
+        echo   "notifications": {
+        echo     "flashWindow": 1
+        echo   },
+        echo   "useSpellChecker": true
+        echo   "showUnreadBadge": true
+        echo }
+      ) > %APPDATA%\Mattermost\config.json
+
+  3. Save the text file with extension .bat. For instance, mattermost-app-install.bat.
```

Comment:
Note from Jeff: You should make it clear that each user has to download and run the batch file themselves. At least, that's the way it's set up here... it can't be used to push the install to multiple users.

Note from Yuya: Yes, but I had thought that admin's deployment system should trigger the process...To install the app for other users, zip version of the app should be used. https://forum.mattermost.org/t/mattermost-client-package/2139/2

Wondering the best way forward?